docker/py3ews/verify.py

```diff
@@ -37,4 +37,4 @@
 assert 'md4' in hashlib.algorithms_available
 hashlib.new('md4', b"text")
 
-print('all is good, `get_localzone() -> {}` is working'.format(test))
+print('all good, `get_localzone() -> {}` is working'.format(test))
```

Comment:
Why do we change it?

Reply:
to satisfy the "update the verify file check" 😂 
please add imports for the relevant packages instead